test/testConstants/PropertyDetailsTestConstants.scala

```diff
@@ -115,7 +127,16 @@ object PropertyDetailsTestConstants {
     firstAccountingPeriodEndDate = None,
     incomeSourceType = Some(foreignIncomeType),
     tradingStartDate = Some(testStartDate),
-    cessation = Some(testCessation)
+    cessation = Some(testPropertyCessation)
+  )
+
+  val ceasedForeignPropertyDetailsCessation2 = PropertyDetailsModel(
```

Comment:
naming convention suggestion, this can be something like:
ceasedForeignPropertyDetailsCessation2 => ceasedForeignPropertyDetailsCessationNoEndDate